pyvrp/tests/test_Solution.py

```diff
@@ -176,6 +176,18 @@ def test_feasibility():
     assert_(not sol.has_time_warp())
 
 
+def test_feasibility_release_times():
+    data = read("data/OkSmallReleaseTimes.txt")
+
+    # Client 1 is released at 20000, but client 2 time window ends at 19500.
+    sol = Solution(data, [[1, 2], [3], [4]])
+    assert_(not sol.is_feasible())
```

Comment:
Can we test that that incurs exactly as much time warp as we would expect from leaving depot at 20K, driving to 1, doing service, and driving to 2 compared to client 2's `tw_late`?